lib/Doctrine/ORM/Tools/EntityGenerator.php

```diff
@@ -1380,15 +1432,26 @@ protected function generateEntityStubMethod(ClassMetadataInfo $metadata, $type,
             $methodTypeHint =  '\\' . $typeHint . ' ';
         }
 
+        $methodReturnType = null;
+        if ($this->strictTypes) {
```

Comment:
I understand. 

Is it ok if a add a specific option for 'strict_types' and another one for return types and scalar type hinting ?